derrida/interventions/management/commands/annotation_data.py

```diff
@@ -94,7 +94,7 @@ def annotation_data(self, intervention):
             # get localized piffle image api client for this canvas
             img = self.localize_iiif_image(intervention.canvas, intervention.work_instance)
             # request 500 width image and convert to str for iiif image url
-            page_iiif = str(img.size(width=500))
+            page_iiif = str(img)
```

Comment:
I think we should still use the 500 width version — it's big enough to be readable 

(if not, need to update/remove the comment)

Reply:
Nope! It's big enough. I'll re-introduce.

Reply:
✅ 4240ce5ac8704c81597e41767d753069b63b1606